privacy_partner_report/report/privacy_partner_xlsx.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+from openerp.report import report_sxw
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from openerp.addons.report_xlsx.report.report_xlsx import ReportXlsx
+except ImportError:
+    _logger.debug("report_xlsx not installed, Excel export non functional")
+
+    class ReportXlsx(object):
+        def __init__(self, *args, **kwargs):
+            pass
+
+
+class ReportPartnerXlsx(ReportXlsx):
+
+    def _search_longest_row(self, tables):
+        res = 0
+        for table in tables:
+            for model in tables[table]:
+                if len(tables[table][model]) > 0:
+                    if len(tables[table][model][0]) > res:
+                        res = len(tables[table][model][0])
+        return res
+
+    def generate_xlsx_report(self, workbook, data, objects):
+        partner = data['form'].get('partner_id', False)
+        partner = self.env['res.partner'].sudo().browse(partner[0])
+        workbook.set_properties({
+            'comments': 'Created with Python and XlsxWriter from Odoo 9.0'})
```

Comment:
I think this is meaningless. Besides, it adds maintenance burden to bump the odoo version.

Please remove it, or at least remove the version.